scripts/dockerbot/pytestbase/get_keys.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python2
+#!/cfme_pristine_venv/bin/python2
```

Comment:
this is indeed for inside of docker, as far as i understood, setup.txt isnt well suited for running in a folder